app/views/coronavirus_local_restrictions/_devolved_nation.html.erb

```diff
@@ -13,12 +13,16 @@
     title: title
   } %>
   <p class="govuk-body">
-    <%= t("coronavirus_local_restrictions.results.match") %> <strong><%= postcode %></strong> to <strong><%= location.country %></strong>.
+    <%= t("coronavirus_local_restrictions.results.match") %> <strong><%= postcode %></strong> to <strong><%= location_lookup.lower_tier_area_name %></strong>.
```

Comment:
More a query than a code concern, Is it possible that this area name doesn't return anything? I know we look up using: 

https://github.com/alphagov/collections/blob/15a5df7af75faa265d62063220a7685915970497/app/services/location_lookup_service.rb#L4

and I wondered if Mapit pretty much guarantees a result to one of those or if we have cases where it may be nil and we should fallback to the country name?

Reply:
I "borrowed" this list from [find-local-council](https://github.com/alphagov/frontend/blob/master/app/controllers/find_local_council_controller.rb#L8-L10). I've tested various supermarkets in the devolved nations and it looks to be ok. From what I can tell, if the postcode matches in Mapit, it will have one of these area codes.

Reply:
👍  Cool cool - looks like it'll only be an empty string rather than a crash if that was to occur